src/application/jobs/qualifier-actions.job.handler.db.ts

```diff
@@ -32,41 +32,64 @@ export class QualifierActionsJobHandler extends JobHandler<Job> {
     let nbFailuresQualification = 0
     let nombreActionsQualifiees = 0
 
+    let offset = 0
+    let actionsAQualifierSQL = []
+
     try {
-      const actionsInProgress = await ActionSqlModel.findAll({
-        where: actionsNonAnnuleesAvecDateEcheanceSuperieureA4Mois(maintenant)
-      })
+      do {
+        actionsAQualifierSQL = await ActionSqlModel.findAll({
+          where: {
+            statut: {
+              [Op.in]: [
```

Comment:
`!== CANCELED` plutôt que `IN ...` ?

Reply:
j'ai voulu etre explicite, on sait jamais si demain on rajoute un autre statut